raiden-dapp/src/components/AddressInput.vue

```diff
@@ -161,6 +156,8 @@ export default class AddressInput extends Mixins(BlockieMixin) {
         });
     }, 800) as unknown) as number;
   }
+
+  paste() {}
```

Comment:
Shouldn't this somehow call the clipboard API?

Reply:
Yes, it should I spend some time trying to get it working, but I was getting some errors with Firefox when attempting to request the clipboard permission. The idea was to not block the PR on the paste, and instead open a new issue about the paste implementation.